src/app/App.js

```diff
@@ -11,16 +11,17 @@ function App() {
         <AuthProvider>
             <BrowserRouter>
                 <Routes>
+                    <Route element={<PrivateRoute />}>
+                        <Route
+                            exact
+                            path='/'
+                            element={<Home />}
+                        />
+                    </Route>
```

Comment:
I liked the nested format better. Found out about it while searching Stack Overflow